### PR TITLE
Add support for Telegram

### PR DESCRIPTION
**Pull request by nimesh-notifi**

### CODE REVIEW

File: packages/notifi-axios-adapter/lib/fragments/supportedTargetTypesFragment.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const supportedTargetTypesFragment = `
+fragment supportedTargetTypesFragment on SupportedTargetTypesForDapp {
+
```
Comment (juni-notifi) [inline packages/notifi-axios-adapter/lib/fragments/supportedTargetTypesFragment.ts:3]: intentionally fetching nothing for now?

File: packages/notifi-core/lib/operations/GetSupportedTargetTypesForDapp.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { ParameterLessOperation, TargetType } from '../models';
+
+export type GetSupportedTargetTypesResult = ReadonlyArray<TargetType>;
+
+export type GetSupportedTargetTypesForDappService = Readonly<{
+  getSupportedTargetTypesForDapp: ParameterLessOperation<GetSupportedTargetTypesResult>;
```
Comment (juni-notifi) [inline packages/notifi-core/lib/operations/GetSupportedTargetTypesForDapp.ts:6]: i don't think we should use ParameterLessOperation, we should take a ClientGetSupportedTargetTypesForDappInput as an input right?

Comment (nimesh-notifi) [reply packages/notifi-core/lib/operations/GetSupportedTargetTypesForDapp.ts:6]: yup, I missed that

File: packages/notifi-axios-adapter/lib/fragments/supportedTargetTypesFragment.ts (hunk 1 above)
Comment (nimesh-notifi) [reply packages/notifi-axios-adapter/lib/fragments/supportedTargetTypesFragment.ts:3]: nope, definitely not my intention!

Comment (nimesh-notifi) [reply packages/notifi-axios-adapter/lib/fragments/supportedTargetTypesFragment.ts:3]: query supportedTargets {
  supportedTargetTypesForDapp(getSupportedTargetTypesForDappInput: {
    dappAddress: ""
  }) {
    
  }

**Pull request merged**